apis/cache/v1alpha2/zz_generated.managed.go

```diff
@@ -33,6 +33,11 @@ func (mg *ReplicationGroup) GetClaimReference() *corev1.ObjectReference {
 	return mg.Spec.ClaimReference
 }
 
+// GetCondition of this ReplicationGroup.
```

Comment:
Would be nice to move these code generation updates into a separate commit if it's not too difficult to do so.